README.md

```diff
@@ -1,24 +1,250 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby should be installed.
+- You need a Terminal.
+
+Then run the command example bellow to install rails.-
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Khaled-AlKhateeb/blog-rails.git
+```
+
+### Install
+
+Install this project with:
+
+
+Example command:
+
+```sh
+  cd blog-rails/
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  gem install rails rspec
+  rails spec spec/models/
+```
+<!--
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Khaled Al-Khateeb**
+
+- GitHub: [@Khaled-AlKhateeb](https://github.com/Khaled-AlKhateeb)
+- Twitter: [@KhaledA93751489](https://twitter.com/KhaledA93751489)
+- LinkedIn: [khaled-al-khateeb](https://www.linkedin.com/in/khaled-al-khateeb-3a1013247/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+Some of the comming futures for this project would be:
+
+- [ ] **Adding User Login**
+- [ ] **Add API endpoints**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Khaled-AlKhateeb/blog-rails/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project, kindly give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+<!--
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- FAQ (optional) -->
+<!--
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please fix the link to [MIT license](https://choosealicense.com/licenses/mit/)
You can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).

Reply:
LICENSE.md file merged to the dev branch, thanks for the advice

Reply:
- [ ] This issue is still present, will you fix the link please? When clicking on this link we are getting 404 page
![Screen Shot 2023-02-01 at 20 15 54](https://user-images.githubusercontent.com/56886064/216128509-5a307790-02e1-46bb-9567-7eafd956a54b.png)